internal/datastore/spanner/watch.go

```diff
@@ -164,3 +181,175 @@ func contextualizedCaveatFromValues(values map[string]any) (*core.Contextualized
 	}
 	return nil, nil
 }
+
+const maxSchemaWatchRetryCount = 15
+
+func (sd spannerDatastore) WatchSchema(ctx context.Context, afterRevision datastore.Revision) (<-chan *datastore.SchemaState, <-chan error) {
+	updates := make(chan *datastore.SchemaState, 10)
+	errs := make(chan error, 1)
+
+	lastRevision := afterRevision
+	retryCount := -1
+	go (func() {
+		running := true
+		for running {
+			retryCount++
+			retryHistogram.Observe(float64(retryCount))
```

Comment:
Why not a gauge or a count(1)? This number is basically monotonically increasing over time, what kind of graph were you planning to plot?